README.md

```diff
@@ -286,6 +286,11 @@ repositories {
 </p>
 </details>
 
+
+## R8 / Proguard
+The specific rules are [already bundled](mordant/src/jvmMain/resources/META-INF/proguard/mordant.pro) into the Jar which can be interpreted by R8 automatically
+
+
```

Comment:
Let's remove this docs section if we're bundling the rules

Reply:
279fbf27fc186f55491b5fc0b7fecabba2262320